templates/index.html

```diff
@@ -325,6 +328,43 @@ <h3 class="p-heading--2">Simple, secure and scalable IoT</h3>
         </div>
       </div>
     </div>
+    <div tabindex="0" role="tabpanel" id="Automotive-tab" aria-labelledby="Automotive" aria-hidden="false">
+      <div class="p-block">
```

Comment:
`p-block` is [deprecated](https://vanillaframework.io/docs/patterns/section#shallow-sections), should be `p-section--shallow` now